src/zjs_sensor.c

```diff
@@ -49,6 +49,7 @@ typedef struct sensor_handle {
 
 static sensor_handle_t *accel_handles = NULL;
 static sensor_handle_t *gyro_handles = NULL;
+static sensor_handle_t *light_handles = NULL;
```

Comment:
zjs_sensor are for all the w3c sensors, and this implements the w3c sensor api, it's not specific to the grove light, it's more generic light sensor